x-pack/plugins/security_solution/public/dashboards/components/title_badges.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import type { DashboardAPI } from '@kbn/dashboard-plugin/public';
+import React from 'react';
+import { useBadges } from '../hooks/use_badges';
+
+const DashboardTitleBadgesComponent: React.FC<{ dashboardContainer: DashboardAPI }> = ({
+  dashboardContainer,
+}) => {
+  const badges = useBadges({ dashboardContainer });
```

Comment:
Instead of this code for the badges, we should likely provide a badges function from the Dashboard plugin, so that if we add a badge -the new `managed` badge for instance, it shows up properly anywhere the portable Dashboard was used.

Reply:
Good point, after aligning the tool bar location with analytic dashboard, we do not need this component to render bages. I've removed this component.